src/libPMacc/include/eventSystem/Manager.hpp

```diff
@@ -74,9 +74,7 @@ namespace PMacc
 
         EventPool& getEventPool();
 
-        int getCount();
-
-
+        std::size_t getCount();
```

Comment:
Yes: size_t is C compatibility and std::size_t is C++ see [here](http://stackoverflow.com/questions/5813700/difference-between-size-t-and-stdsize-t).
Most of the C++ compilers are not standard compliant by also defining everything outside std when including `cstddef` so this never is a real problem.